sdk/python/foundation-models/system/evaluation/image-object-detection/fridge-eval-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "metrics": ["mean_average_precision", "recall", "precision", "per_label_metrics", "image_level_binary_classsifier_metrics", "confusion_matrices_per_score_threshold"],
+  "class_labels": ["can", "carton", "milk_bottle", "water_bottle"],
```

Comment:
I think we are only reading metrics, iou_threshold and box_score_threshold from this config